tags/health-and-fitness.yml

```diff
@@ -0,0 +1,4 @@
+name: Health and Fitness
+description: '[Medical](https://en.wikipedia.org/wiki/Medical_software), [Health](https://en.wikipedia.org/wiki/Health_information_technology) and [Fitness](https://en.wikipedia.org/wiki/Fitness_tracker) software.'
+related_tags:
+ - Genealogy
```

Comment:
```suggestion
```

I also don't find this relation relevant. Other than that, good to merge, thanks